hermes-console/src/components/navigation-drawer/NavigationItem.vue

```diff
@@ -39,4 +70,21 @@
     text-decoration: none;
     color: inherit;
   }
+  .nav-title {
+    padding-left: 8px;
+    display: block;
+  }
+  .nav-title-link {
+    margin-left: 3%;
+    display: block;
+    width: 100%;
+  }
+  .nav-title-align {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-width: 0;
+    padding-left: 8px;
+    box-sizing: border-box;
+  }
```

Comment:
You can replace most of this css with vuetify classes just like that:
`<v-list-item-title class="d-flex align-center w-100 pl-2">
  ...
</v-list-item-title>`

we are already using this approach in the codebase, and it is more readable